src/main/resources/assets/tfc/lang/ja_jp.json

```diff
@@ -120,6 +120,12 @@
   "subtitles.entity.tfc.rat.hurt": "Rat squeals",
   "subtitles.entity.tfc.rat.step": "Rat patters",
   "subtitles.entity.tfc.rooster.cry": "Rooster calls",
+  "subtitles.entity.tfc.dog.ambient": "Wolf Barks",
```

Comment:
dog barks?

Reply:
I've changed this so that wolves and dogs just have separate subtitles.